packages/back-end/src/integrations/SqlIntegration.ts

```diff
@@ -473,6 +475,45 @@ export default abstract class SqlIntegration
     });
   }
 
+  formatInformationSchema(results: InformationSchema[]) {
```

Comment:
Open to feedback on a better way to do this as it's very verbose. Essentially, the results come in with a unique row per catalog, data_set, table, and column, and we need a single array where each index is a specific data_set, and that data_set should have an array of tables, and each table, should have an array of columns and their associated data type.

```
sampleInput = [
   {
        table_name: "events",
        column_name: "timestamp",
        data_type: "timestamp without time zone",
        table_catalog: "gb"
   },
   {
        table_name: "events",
        column_name: "value",
        data_type: "integrer",
        table_catalog: "gb"
   },
   {
        table_name: "events",
        column_name: "browser",
        data_type: "string",
        table_catalog: "gb"
   },
   {
        table_name: "sessions",
        column_name: "userid",
        data_type: "string",
        table_catalog: "gb"
   },
]
```

And a sample output would look something like this.

```
 sampleOutput = [
 {
     catalog_name = "gb",
     tables = [
         {
              table_name: "events",
              columns: [
                   {
                        column_name: "timestamp",
                        data_type: "timestamp without time zone"
                   }, 
                   {
                        column_name: "value",
                        data_type: "integer"
                   },
                   {
                        column_name: "browser",
                        data_type: "string"
                   }, 
              ]
          },
          {
              table_name: "sessions",
              columns: [
                  {
                        column_name: "userId",
                        data_type: "integer"
                   }, 
              ]
          },
     ]
 },
]
```